### ISSUE TITLE
Wait spinner doesn't appear on startup or when loading a track

### ISSUE THREAD

**Issue by jrobinso** [igvteam/igv.js#1361]
For some track types,  neither track nor wait cursor appear until the track data is fully loaded.   This is possibly true of all track types, but is very noticeable with the example. ```dev/maf/maf_tcga.html```.  Nothing happens for around 5-10 seconds after starting it, after which the track appears fully loaded.   Compare this to behavior prior to the "dom refactor" commit (10725d29),  a spinner appears immediately, after the data is loaded the track appears.

Note,  if checking earlier versions the maf_tcga.html file is in the dev/ directory, not dev/maf.   Also, you need to delete "igv-utils" from node_modules, and package-lock.json, and ```npm install``` as the version of igv-utils differs.

Also, the look (icon) of the spinner has changed.   Not sure if that was deliberate or not.

**Comment by jrobinso** [comment]
The same problem exists if loading a track from a URL,  for example try loading this by url from igv.org/app-test

```
https://s3.amazonaws.com/igv.org.demo/TCGA.BRCA.mutect.995c0111-d90b-4140-bee7-3845436c3b42.DR-10.0.somatic.maf.gz
```

**Comment by jrobinso** [comment]
OK I found the cause of this.  Track classes have an optional "postInit" function that is called after track creation from "loadTrack"from what is now(as I type) at line 792.   This function can take a lot of time, particularly for large non-indexed files.  Basically the entire file is read.   There used to be a spinner wrapping "loadTrack", started at the top and closed in a finally block.   This was removed, thus no spinner.
